scripts/mafia.js

```diff
@@ -8,14 +8,14 @@
 // Global variables inherited from scripts.js
 /*global mafiabot, getTimeString, updateModule, script, sys, SESSION, sendChanAll, require, Config, module, sachannel, staffchannel, sendChanHtmlAll, isSuperAdmin*/
 /*jshint laxbreak:true,shadow:true,undef:true,evil:true,trailing:true,proto:true,withstmt:true,eqnull:true*/
-var MAFIA_CHANNEL = "Mafia";
 
+var MAFIA_CHANNEL = "Mafia";
 var is_command = require("utilities.js").is_command;
 var nonFlashing = require("utilities.js").non_flashing;
 var html_escape = require("utilities.js").html_escape;
 
 function Mafia(mafiachan) {
-    this.version = "2016-10-22";
+    this.version = "2016-10-25h";
```

Comment:
When I test it helps me know if the script actually updated cause sometimes GitHub sucks =x